docs/examples/single_moving_target_relative_Doppler.ipynb

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    vmax = lambd / (4*Ts_s)
</pre><p>Please use v_max (note the underscore). This is suggested in https://peps.python.org/pep-0008 It is ok to sometimes deviate from pep8 (for example it can be useful to use capitalized variable names like Ts_s)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rf-eng/scikit-radar/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>